recipes/xsel/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -ex
+
+# Cf. https://github.com/conda-forge/staged-recipes/issues/673, we're in the
+# process of excising Libtool files from our packages. Existing ones can break
+# the build while this happens. We have "/." at the end of $PREFIX to be safe
+# in case the variable is empty.
+find $PREFIX/. -name '*.la' -delete
+
+echo libtoolize
+libtoolize
+echo aclocal -I $PREFIX/share/aclocal -I $BUILD_PREFIX/share/aclocal
+aclocal -I $PREFIX/share/aclocal -I $BUILD_PREFIX/share/aclocal
+echo autoconf
+autoconf
+echo automake --force-missing --add-missing --include-deps
+automake --force-missing --add-missing --include-deps
```

Comment:
You already have `set -x` at the top, so all these echos are redundant.

Reply:
Done.